R/unbump-version.R

```diff
@@ -19,7 +19,8 @@ unbump_version_impl <- function() {
 
   parent_commit_id <- gert::git_commit_info(last_commit$commit)$parent
 
-  cli_alert_success("Resetting to parent commit {.field {parent_commit_id}}.")
+  message_id <- Sys.getenv("FLEDGE_UNBUMP_TEST_COMMIT", parent_commit_id)
```

Comment:
if there were a fledge.quiet option, I'd remove this.

Reply:
The commit ID should be stable if we fix committer user + time?

Reply:
Ouch, I hadn't used the author and committer arguments in enough places, so this PR does a bit of refactoring to fix that.